Option to include stride and device annotation in gm.print_readable()

SherlockNoMad opened the pull request:
Summary:
Sample output for gm.print_readable(include_stride=True, include_device=True)

```
        getitem_21: "i32[1200][1]cuda:0" = auto_functionalized_4[1]
        copy_2: "f32[2, 60][60, 1]cuda:1"  = ....
```

Test Plan: CI

Differential Revision: D55949129

Chillee commented on torch/fx/graph.py:
```diff
@@ -387,6 +389,8 @@ def _gen_python_code(
 
         # Wrap string in list to pass by reference
         maybe_return_annotation : List[str] = ['']
+        include_stride = include_stride or (os.environ.get("ANNOTATE_STRIDE", "0") == "1")
```
Perhaps call this something like FX_GRAPH_VERBOSE (it would be good to have FX_GRAPH in the env variable name)

Chillee reviewed (approved):
tests failing, but lgtm.

SherlockNoMad committed:
Option to include stride and device annotation in gm.print_readable() (#123690)

Summary:

Sample output for gm.print_readable(include_stride=True, include_device=True)

```
        getitem_21: "i32[1200][1]cuda:0" = auto_functionalized_4[1]
        copy_2: "f32[2, 60][60, 1]cuda:1"  = ....
```

Test Plan: CI

Reviewed By: Chillee

Differential Revision: D55949129
